tensorrt_llm/_torch/auto_deploy/transform/library/fuse_rope_attention.py

```
@@ -0,0 +1,434 @@
+    def _remove_position_ids_placeholder(self, gm: GraphModule) -> bool:
+        graph = gm.graph
+        position_ids_node = graph.find_nodes(op="placeholder", target="position_ids")[0]
+        input_ids_node = graph.find_nodes(op="placeholder", target="input_ids")[0]
+
+        # Get size from token_ids placeholder instead of position_ids placeholder
+        sym_size_int_nodes = graph.find_nodes(
+            op="call_function", target=torch.ops.aten.sym_size.int
+        )
+        for node in sym_size_int_nodes:
+            if node.args[0] == position_ids_node:
+                new_args = (input_ids_node,) + node.args[1:]
```

Ruff (0.14.8)

Consider `(input_ids_node, *node.args[1:])` instead of concatenation

Replace with `(input_ids_node, *node.args[1:])`

(RUF005)